test/e2e-v2/cases/php/expected/service-instance-php.yml

```diff
@@ -16,7 +16,7 @@
 # under the License. shadow
 
 {{- contains .}}
-- id: {{ b64enc "php" }}.1_{{ b64enc "php-instance" }}
```

Comment:
Is the php instance predictable or not?

Reply:
the new version of the php agent does not support fixed instance name.

Reply:
@jmjoy Could you confirm here? Is the php agent using the UUID or something?